src/System.IO.Pipelines.Extensions/System.IO.Pipelines.Extensions.csproj

```diff
@@ -12,6 +12,6 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="$(CoreFxStableVersion)" />
-    <PackageReference Include="System.Numerics.Vectors" Version="$(CoreFxStableVersion)" />
+    <PackageReference Include="System.Numerics.Vectors" Version="$(SystemNumericsVectorsVersion)" />
```

Comment:
@davidfowl, is that ok? I

Reply:
We were already referencing the latest System.Numerics.Vectors package prior to https://github.com/dotnet/corefxlab/pull/1736.

Reply:
Hmm, not sure yet.